src/components/Home/Home.tsx

```diff
@@ -1,25 +1,35 @@
 import React from 'react';
 
-import ShortInfoCard from '../ShotrInfoCard/ShortInfoCard';
+import { DetailedCardComponent } from '../DetailedCardComponent/DetailedCardComponent';
+import ShortInfoCard from '../ShortInfoCard/ShortInfoCard';
+
 import { SettingsIcon } from '../icons/icons';
 import { MusicType } from '../../enums';
 
+import './home.scss';
 const Home = () => {
```

Comment:
here we can remove curly braces + return because of no logic inside component and we use => arrow function